pydoctor/driver.py

```diff
@@ -251,6 +258,10 @@ def main(args: Sequence[str] = sys.argv[1:]) -> int:
     if options.configfile:
         readConfigFile(options)
 
+    if options.enable_intersphinx_cache_deprecated:
+        print("The --enable-intersphinx-cache option is deprecated; "
+              "the cache is now enabled by default.", file=sys.stderr)
```

Comment:
maybe log it as a deprecation warning, instead of a simple print... the idea that a deprecation warning should be picked by an automated test run.... but since this is CLI usage... I don't know. Maybe the stderr print will also do the job.

```suggestion
        warnings.warn("The --enable-intersphinx-cache option is deprecated; "
              "the cache is now enabled by default.", DeprecationWarning)
```

Reply:
`DeprecationWarning` is intended to inform developers that they're using deprecated API without bothering the user about it (since warnings are disabled by default). But since this is a CLI option that is deprecated, we should be alerting the user to update their scripts.